flow360/__init__.py

```diff
@@ -92,6 +92,8 @@
     LinearSolver,
     NavierStokesSolver,
     PressureCorrectionSolver,
+    TurbulenceModelConstantsSA,
```

Comment:
Can we use a better name here? SpalartAllmarasModelConstants? similarly for kOmegaSST we can use KOmegaSSTModelConstants

Reply:
Done and renamed to `KOmegaSSTModelConstants` and `SpalartAllmarasModelConstants`